hbase-compression/hbase-compression-zstd/src/test/java/org/apache/hadoop/hbase/io/compress/zstd/TestZstdByteBuffDecompressor.java

```diff
@@ -40,10 +40,14 @@ public class TestZstdByteBuffDecompressor {
   public static final HBaseClassTestRule CLASS_RULE =
     HBaseClassTestRule.forClass(TestZstdByteBuffDecompressor.class);
 
-  // "HBase is awesome" compressed with zstd, and then prepended with metadata as a
-  // BlockCompressorStream would
-  private static final byte[] COMPRESSED_PAYLOAD =
-    Bytes.fromHex("000000100000001928b52ffd2010810000484261736520697320617765736f6d65");
+  /*
+   * "HBase is fun to use and very fast" compressed with zstd, and then prepended with metadata as a
+   * BlockCompressorStream would. The phrase is split in three parts and put into the payload in
+   * this structure: (block 1: (chunk 1: HBase is), (chunk 2: fun to use)), (block 2: (chunk 1: and
+   * very fast))
+   */
+  private static final byte[] COMPRESSED_PAYLOAD = Bytes.fromHex(
+    "000000130000001228b52ffd20094900004842617365206973200000001428b52ffd200b59000066756e20746f20757365200000000d0000001628b52ffd200d690000616e6420766572792066617374");
```

Comment:
nit: should we break this up to appease checkstyle?

Reply:
tbh if spotless is okay with this, I am too

Reply:
Yeah, might as well split the long line. Then someone else won't have to clean it up later.

Reply:
No problem, done